templates/bionomia.md

```diff
@@ -1,20 +1,28 @@
 ---
 {{#if orcid}}ORCID: {{orcid}}{{/if}}
 {{#if wikidata}}Wikidata: {{wikidata}}{{/if}}
-aliases: ["{{{query}}}"]
+uri: {{uri}}
+aliases: [{{other_names}}]
```

Comment:
Element values in `other_names` may include commas, which are the delimiters separating the different entries in the `aliases` list. So each of the values from `other_names` will need to be quoted: ie something like:
```
aliases: [{{#if other_names}}{{#each other_names}} "{{this}}",{{/each}}{{/if}} "{{query}}"]
```
This snippet also adds the query value as discussed above.